classes/class-gf-securesubmit.php

```diff
@@ -755,7 +757,7 @@ public function validation($validation_result)
             }
             if (in_array($fieldType, $this->ccFields) && $field_on_curent_page) {
                 $this->isCC = $field;
-                if (empty($this->validateACH()) && $this->getSecureSubmitJsError() && $this->hasPayment($validation_result)) {
+                if (false == $this->validateACH() && $this->getSecureSubmitJsError() && $this->hasPayment($validation_result)) {
```

Comment:
That being said, I can replace this with a two part solution, create a temporary variable then use `empty` as before